packages/qscintilla/build.sh

```diff
@@ -0,0 +1,41 @@
+TERMUX_PKG_HOMEPAGE=https://riverbankcomputing.com/software/qscintilla
+TERMUX_PKG_DESCRIPTION="QScintilla is a port to Qt of the Scintilla editing component"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="Simeon Huang <symeon@librehat.com>"
+TERMUX_PKG_VERSION=2.12.1
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL="https://www.riverbankcomputing.com/static/Downloads/QScintilla/${TERMUX_PKG_VERSION}/QScintilla_src-${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=a7331c44b5d7320cbf58cb2382c38857e9e9f4fa52c405bd7776c8b6649836c2
+TERMUX_PKG_DEPENDS="qt5-qtbase"
+# qttools is only needed to build Qt Designer's plugins
+TERMUX_PKG_BUILD_DEPENDS="qt5-qtbase-cross-tools, qt5-qttools"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_NO_STATICSPLIT=true
+
+termux_step_configure () {
+    for i in src designer; do
+        cd "${TERMUX_PKG_SRCDIR}/${i}" && {
+            "${TERMUX_PREFIX}/opt/qt/cross/bin/qmake" \
+                -spec "${TERMUX_PREFIX}/lib/qt/mkspecs/termux-cross"
+        }
+    done
+    unset i
+}
+
+termux_step_make() {
+    cd "${TERMUX_PKG_SRCDIR}/src" && {
+        make -j "${TERMUX_MAKE_PROCESSES}"
+    }
+}
+
+termux_step_make_install() {
+    cd "${TERMUX_PKG_SRCDIR}/src" && {
+        make install
+    }
+
+    cd "${TERMUX_PKG_SRCDIR}/designer" && {
+        make -j "${TERMUX_MAKE_PROCESSES}"
+        make install
+    }
```

Comment:
This can be put in termux_step_post_make_install, and termux_step_make_install then not overridden

Reply:
It still needs to go inside the subdirectories no? Is the default script smart enough to go one level down?

Reply:
No, it is not, but moving `cd ${TERMUX_PKG_SRCDIR}/designer; make -j "${TERMUX_MAKE_PROCESSES}; make install"` to `termux_step_post_make_install` would be preferable so that we do not need to override the default function.

By not overriding it becomes easier to maintain, so that we can add for example a new arg in just one place and not in all individual build.sh scripts.